hicommonwealth/commonwealth
# Show networks alphabatically stake wise sorted in network selection dropdown during create community flow

…ropdown



## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/6765

## Description of Changes
- Chain Networks now show up alphabetically sorted in Create Community flow
- Chain Networks with stake enabled will show up first with a 'Community Stake' label

## "How We Fixed It"
N/A

## Test Plan
- Go to step 2 of Create Community flow
- Open the chain selection dropdown and verify you see the chains sorted correctly i.e alphabetically sorted + those with stake enabled will come first with a 'Community Stake' label

## Deployment Plan
N/A

## Other Considerations
N/A
